js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -1702,7 +1702,10 @@ function processQuery(query, explain, planIndex) {
         }
         if (node.sortmode !== 'unset') {
           gatherAnnotations.push('sort mode: ' + node.sortmode);
-        } 
+        }
+        if (node.elements.length === 0) {
+          gatherAnnotations.push('unsorted');
+        }
```

Comment:
Is it possible that the sort mode is set to "sorted" but that there are 0 elements, so we will end up with `sort mode: sorted, unsorted` later?

Reply:
No, the decision whether `sortmode` will be set is exactly based on whether `elements` is empty or not. So this should be correct.

Reply:
Ok